recipes/tests.py

```diff
@@ -61,3 +62,188 @@ def test_same_name_ingredient(self):
     def test_list_ingredient(self):
         response = self.list_ingredient()
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+
+class ListCreateRecipeTestCase(TestCase):
+    url = reverse('list-create-recipe')
+
+    def create_user(self, username='testuser', email='testuser@mail.com'):
```

Comment:
You use this method in a lot of `TestCase`, you can wite a `mixin class` for this.